PR failed on sandbox during `post_deploy_tdr`

https://gitlab.dev.singlecell.gi.ucsc.edu/ucsc/azul/-/jobs/133091

```
Traceback (most recent call last):
  File "/builds/ucsc/azul/scripts/post_deploy_tdr.py", line 125, in <module>
    main()
    ~~~~^^
  File "/builds/ucsc/azul/scripts/post_deploy_tdr.py", line 120, in main
    validator.verify_sources()
    ~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/builds/ucsc/azul/scripts/post_deploy_tdr.py", line 89, in verify_sources
    raise e
  File "/usr/local/lib/python3.13/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/builds/ucsc/azul/scripts/post_deploy_tdr.py", line 96, in verify_source
    ref = plugin.resolve_source(source_spec)
  File "/builds/ucsc/azul/src/azul/plugins/__init__.py", line 716, in resolve_source
    id = self._lookup_source_id(spec)
  File "/builds/ucsc/azul/src/azul/plugins/repository/tdr.py", line 182, in _lookup_source_id
    return self.tdr.lookup_source(spec)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/builds/ucsc/azul/src/azul/terra.py", line 384, in lookup_source
    source = self._lookup_source(source_spec)
  File "/builds/ucsc/azul/src/azul/terra.py", line 417, in _lookup_source
    raise self._insufficient_access(str(endpoint))
azul.RequirementError: The service account (SA) 'azul-ucsc-0-sandbox@platform-hca-dev.iam.gserviceaccount.com' is not authorized to access https://jade.datarepo-dev.broadinstitute.org/api/repository/v1/snapshots?filter=hca_dev_edc54c5d82404681844462a086d1d1be__20240410_dcp2_20240410_dcpPilot&limit=2 or that resource does not exist. Make sure that it exists, that the SA is registered with SAM and has been granted read access to the resource.
make: *** [Makefile:114: auto_deploy] Error 1
```

--- a/UPGRADING.rst
+++ b/UPGRADING.rst
@@ -20,6 +20,23 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#7128 Add support for HCA atlas (partial)
+=========================================
+
+In your personal HCA deployments, add the following snapshots to the ``dcp3``
+catalog:
+
+- ``8c4e43fb…``
+- ``8f1f653d…``
+- ``aca93e28…``
+- ``c0518445…``
+- ``c4e11369…``
+- ``edc54c5d…``
+
+As always, use the sandbox deployment's ``environment.py`` as a model when
+upgrading personal deployments.
+
+
 #7617 Index one snapshot from upcoming lm10 release in `dev`
 ============================================================
 

--- a/deployments/dev/environment.py
+++ b/deployments/dev/environment.py
@@ -168,17 +168,23 @@ dcp2_sources = union({}, 105, delta([
     source('bigquery', 'datarepo-dev-96d8e08c', 'hca_dev_faeedcb0e0464be7b1ad80a3eeabb066__20210831_20210903'),
 ]))
 
-dcp3_sources = union(dcp2_sources, 105, delta([
+dcp3_sources = union(dcp2_sources, 111, delta([
     # @formatter:off
     source('bigquery', 'datarepo-dev-3d6d24ad', 'hca_dev_05657a599f9d4bb9b77b24be13aa5cea__20210827_20230215'),
     source('bigquery', 'datarepo-dev-76de829d', 'hca_dev_2043c65a1cf84828a6569e247d4e64f1__20210831_20210907', pop),
     source('bigquery', 'datarepo-dev-b46086a9', 'hca_dev_51f02950ee254f4b8d0759aa99bb3498__20210827_20230215'),
     source('bigquery', 'datarepo-dev-8a52f620', 'hca_dev_6307bc2662764b85bd26c403d52b6eb2__20251110_dcp2_20251110_dcp99'),  # noqa E501
+    source('bigquery', 'datarepo-dev-fba5d628', 'hca_dev_8c4e43fbf7f344468367a3fe9e0b8fd7__20250828_dcp2_20250829_dcp53'),  # noqa E501
+    source('bigquery', 'datarepo-dev-857f1dfe', 'hca_dev_8f1f653d3ea14d8eb4a7b97dc852c2b1__20230815_dcp2_20250829_dcp53'),  # noqa E501
+    source('bigquery', 'datarepo-dev-55e5c791', 'hca_dev_aca93e287d874aa4b8ae498b9b235f46__20250730_dcp2_20250801_dcp52'),  # noqa E501
+    source('bigquery', 'datarepo-dev-e9a7254b', 'hca_dev_c05184453b3b49c6b8fcc41daa4eacba__20220213_dcp2_20250514_dcp49'),  # noqa E501
+    source('bigquery', 'datarepo-dev-71abe9ae', 'hca_dev_c4e1136978d44d29ba8eb67907c4c65c__20220630_dcp2_20241107_dcp44'),  # noqa E501
     source('bigquery', 'datarepo-dev-27c50fbc', 'hca_dev_c5f4661568de4cf4bbc2a0ae10f08243__20210827_20230215'),
     source('bigquery', 'datarepo-dev-2a4ab485', 'hca_dev_d2111fac3fc44f429b6d32cd6a828267__20210830_20230215'),
     source('bigquery', 'datarepo-dev-1005632d', 'hca_dev_d3ac7c1b53024804b611dad9f89c049d__20210827_20230215'),
     source('bigquery', 'datarepo-dev-8709b362', 'hca_dev_df88f39f01a84b5b92f43177d6c0f242__20210827_20230215'),
     source('bigquery', 'datarepo-dev-e1712bfa', 'hca_dev_e526d91dcf3a44cb80c5fd7676b55a1d__20210902_20230215'),
+    source('bigquery', 'datarepo-dev-1d06e80a', 'hca_dev_edc54c5d82404681844462a086d1d1be__20240410_dcp2_20240410_dcpPilot'),  # noqa E501
     # @formatter:on
 ]))
 

--- a/deployments/sandbox/environment.py
+++ b/deployments/sandbox/environment.py
@@ -170,17 +170,23 @@ dcp2_sources = union({}, 105, delta([
     source('bigquery', 'datarepo-dev-96d8e08c', 'hca_dev_faeedcb0e0464be7b1ad80a3eeabb066__20210831_20210903'),
 ]))
 
-dcp3_sources = union(dcp2_sources, 105, delta([
+dcp3_sources = union(dcp2_sources, 111, delta([
     # @formatter:off
     source('bigquery', 'datarepo-dev-3d6d24ad', 'hca_dev_05657a599f9d4bb9b77b24be13aa5cea__20210827_20230215'),
     source('bigquery', 'datarepo-dev-76de829d', 'hca_dev_2043c65a1cf84828a6569e247d4e64f1__20210831_20210907', pop),
     source('bigquery', 'datarepo-dev-b46086a9', 'hca_dev_51f02950ee254f4b8d0759aa99bb3498__20210827_20230215'),
     source('bigquery', 'datarepo-dev-8a52f620', 'hca_dev_6307bc2662764b85bd26c403d52b6eb2__20251110_dcp2_20251110_dcp99'),  # noqa E501
+    source('bigquery', 'datarepo-dev-fba5d628', 'hca_dev_8c4e43fbf7f344468367a3fe9e0b8fd7__20250828_dcp2_20250829_dcp53'),  # noqa E501
+    source('bigquery', 'datarepo-dev-857f1dfe', 'hca_dev_8f1f653d3ea14d8eb4a7b97dc852c2b1__20230815_dcp2_20250829_dcp53'),  # noqa E501
+    source('bigquery', 'datarepo-dev-55e5c791', 'hca_dev_aca93e287d874aa4b8ae498b9b235f46__20250730_dcp2_20250801_dcp52'),  # noqa E501
+    source('bigquery', 'datarepo-dev-e9a7254b', 'hca_dev_c05184453b3b49c6b8fcc41daa4eacba__20220213_dcp2_20250514_dcp49'),  # noqa E501
+    source('bigquery', 'datarepo-dev-71abe9ae', 'hca_dev_c4e1136978d44d29ba8eb67907c4c65c__20220630_dcp2_20241107_dcp44'),  # noqa E501
     source('bigquery', 'datarepo-dev-27c50fbc', 'hca_dev_c5f4661568de4cf4bbc2a0ae10f08243__20210827_20230215'),
     source('bigquery', 'datarepo-dev-2a4ab485', 'hca_dev_d2111fac3fc44f429b6d32cd6a828267__20210830_20230215'),
     source('bigquery', 'datarepo-dev-1005632d', 'hca_dev_d3ac7c1b53024804b611dad9f89c049d__20210827_20230215'),
     source('bigquery', 'datarepo-dev-8709b362', 'hca_dev_df88f39f01a84b5b92f43177d6c0f242__20210827_20230215'),
     source('bigquery', 'datarepo-dev-e1712bfa', 'hca_dev_e526d91dcf3a44cb80c5fd7676b55a1d__20210902_20230215'),
+    source('bigquery', 'datarepo-dev-1d06e80a', 'hca_dev_edc54c5d82404681844462a086d1d1be__20240410_dcp2_20240410_dcpPilot'),  # noqa E501
     # @formatter:on
 ]))
 

